### PR TITLE
fix: removed onhover show from View Switcher Lock Design

### LINKED ISSUE #4653: View Switcher Lock Design
### Current Behavior

The lock is displayed on hover.

https://github.com/twentyhq/twenty/assets/19412894/15286239-4523-4592-8ec3-52b22158e9f6

### Desired Behavior

The lock should always be visible. Its color should be `Text/Tertiary`

CleanShot 2024-03-26 at 10 00 33

### PR DESCRIPTION
**Pull request by HrushiYadav**
Fix #4653

### CODE REVIEW

**Comment by Kanav-Arora** [comment]
Hi @HrushiYadav 
Thanks for contributing!

There should be a minor change. In the below screenshot you can see the pencil icon is also static. 
Only the locked view should be static but the pencil icon should be visible only on hover.
Moreover please set the icon color to font.color.tertiary

Screenshot 2024-03-26 at 10 32 40 PM

File: packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/StyledMenuItemBase.tsx [hunk 1]
```diff
@@ -104,18 +104,9 @@ export const StyledHoverableMenuItemBase = styled(StyledMenuItemBase)<{
   isMenuOpen: boolean;
 }>`
   & .hoverable-buttons {
-    pointer-events: none;
-    position: fixed;
+    pointer-events: auto;
+    position: static;
```
Comment (Kanav-Arora) [inline packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/StyledMenuItemBase.tsx:108]: Here and the old code was better which had a hover block

Comment (Kanav-Arora) [inline packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/StyledMenuItemBase.tsx:108]: Here and the old code was better which had a hover block

**Review by Kanav-Arora — CHANGES_REQUESTED** [review]

**Comment by HrushiYadav** [comment]
@Kanav-Arora It will take time as I'm facing some issues with the setup

**Comment by FelixMalfait** [comment]
Closing for now but feel free to re-open if problem isn't fixed by then! Thanks for your attempt to contribute 💙

**Comment by Kanav-Arora** [comment]
@HrushiYadav 
When you'll complete the setup feel free to reopen and continue working on this issue.

**Pull request closed without merging**